src/app/availability/availability-table/availability-table.component.ts

```diff
@@ -0,0 +1,28 @@
+﻿import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-availability-table',
+  templateUrl: './availability-table.component.html',
+  styleUrls: [ './availability-table.component.css' ]
+})
+export class AvailabilityTableComponent implements OnInit {
+
+  sprint = 3;
+  startDate = '10/11/2012';
+  endDate = '13/11/2013';
+
+  comment: string;
+  remainingTime: string;
+  declaredTime: string;
```

Comment:
mocki
no i chyba to nie jest taki format jakiego wymaga api

Reply:
Dałem chwilowo godziny tylko, pewnie w przyszłości jak uwzględnimy kwadranse to dam jakoś inaczej będzie